src/Blocks/BlockButton.h

```diff
@@ -67,7 +67,7 @@ class cBlockButtonHandler :
 			case BLOCK_FACE_XM: return 0x2;
 			case BLOCK_FACE_XP: return 0x1;
 			case BLOCK_FACE_YM: return 0x0;
-			default:
+			case BLOCK_FACE_NONE:
```

Comment:
The problem is that the  Monster type has a couple of special values, and you don't want to treat them the same as the other monsters, and you want the compiler to nag you about all to logic that needs changing every-time you add a monster.

Yes there are some places where this causes a huge annoyance, but we have pragma's to deal with those places, and explicitly say, yes we're ignoring everything else.